src/Examine/Providers/BaseIndexProvider.cs

```diff
@@ -71,13 +71,13 @@ protected ValueSetValidationResult ValidateItem(ValueSet item)
         /// Validates the items and calls <see cref="M:Examine.Providers.BaseIndexProvider.PerformIndexItems(System.Collections.Generic.IEnumerable{Examine.ValueSet})" />
         /// </summary>
         /// <param name="values"></param>
-        public void IndexItems(IEnumerable<ValueSet> values)
+        public virtual void IndexItems(IEnumerable<ValueSet> values)
```

Comment:
If we will do as comments above we dont need override that methods, but I agree they should be virtual anyway :)